app/js/components/LabResultEntry.jsx

```diff
@@ -195,7 +195,7 @@ export class LabResultEntry extends PureComponent {
                 {
                   type: "obs",
                   path: "test-order-number",
-                  concept: CONSTANTS.labResultsTestOrderNumberConcept,
+                  concept: "393dec41-2fb5-428f-acfa-36ea85da6666",
```

Comment:
@rhenshaw56  @rhenshaw56  is there a reason we had to switch to a hard-coded UUID here?